docs/source/markdown/podman-logs.1.md

```diff
@@ -39,6 +39,14 @@ strings (e.g. 10m, 1h30m) computed relative to the client machine's time. Suppor
 time stamps include RFC3339Nano, RFC3339, 2006-01-02T15:04:05, 2006-01-02T15:04:05.999999999, 2006-01-02Z07:00,
 and 2006-01-02.
 
+#### **--until**=*TIMESTAMP*
```

Comment:
Maybe it's my warped mind, but when I saw this flag, I thought it would act like tail on the log for the specified time period, not show the logs from the "last" time period.  

Is there precedence for "until" vs "last" or "prior" or some such?

Reply:
Sadly this is what Docker calls i.
```
docker logs --help
...
      --until string   Show logs before a timestamp (e.g.
                       2013-01-02T13:23:37Z) or relative (e.g. 42m for 42
                       minutes)
```